doc/developers/contributing.rst

```diff
@@ -678,54 +672,6 @@ To test code coverage, you need to install the `coverage
 
 3. Loop.
 
-Developers web site
--------------------
```

Comment:
The wiki seems outdated, and with lots of TODO, so I don't think it's a particularly useful resource. This is a typical rabit-hole risk for new users.

Reply:
Let's put a note on the wiki to that effect.